matter_server/common/helpers/util.py

```diff
@@ -101,6 +103,12 @@ def parse_value(name: str, value: Any, value_type: Type, default: Any = MISSING)
     if isinstance(value_type, str):
         value_type = eval(value_type)
 
+    if isinstance(value, dict):
+        if hasattr(value_type, "from_dict"):
+            return value_type.from_dict(value)
+        if hasattr(value_type, "FromDict"):
+            return value_type.from_dict(value)
```

Comment:
Shouldn't this be? Also, which models have these?
```suggestion
        if hasattr(value_type, "FromDict"):
            return value_type.FromDict(value)
```

Reply:
it's not in use actually

Reply:
Then remove it in a future PR